digdag-standards/src/main/java/io/digdag/standards/operator/state/PollingWaiter.java

```diff
@@ -87,9 +123,20 @@ public PollingWaiter withPollInterval(DurationInterval retryInterval)
     public <T> T await(Operation<Optional<T>> f)
     {
         TaskState pollState = state.nestedState(stateKey);
-
+        Long startTime = pollState.params().get(START_TIME, Long.class, null);
```

Comment:
Digdag's code base highly takes advantage of `java.time` package. `Instant` is highly preferred than System.currentTimeMillis.
* Existent use of `System.currentTimeMillis()`:  https://github.com/treasure-data/digdag/search?q=currentTimeMillis&unscoped_q=currentTimeMillis
* Existent use of `Instant.now()`: https://github.com/treasure-data/digdag/search?q=%22Instant.now%22&unscoped_q=%22Instant.now%22

Reply:
Got it.  Will fix it.